Workbooks/Network Insights/AzureFirewallWorkbooks/NetworkInsights-AzureFirewallMetrics-StructuredLogs/NetworkInsights-AzureFirewallMetricsStructuredLogs.workbook

```diff
@@ -0,0 +1,2670 @@
+{
+    "version": "Notebook/1.0",
+    "items": [
+      {
+        "type": 1,
+        "content": {
+          "json": "## Azure Firewall Workbook\r\n---\r\n"
```

Comment:
nit: is this header adding any information?  people know they are in workbooks already, and the title of the blade would already have the template name, so not sure this adds anything, but does take up important real eastate at the top of the window

Reply:
It is just a title for the workbook

![image](https://github.com/microsoft/Application-Insights-Workbooks/assets/44907284/3bd8f1e7-25d6-4a66-ac98-dc7db45c1d09)

I will bring this to the discussion with my team, but I would like to proceed without changing the title. Is that ok?
Soon I will probably be removing the previous 2 workbooks from the gallery, keeping only the new one.

Reply:
that's true, but look:
![image](https://github.com/microsoft/Application-Insights-Workbooks/assets/10158007/5da1066c-b8ee-4a5c-9b87-8cd2d2217123)

the word azure firewall appears 10x already on the screen?

Reply:
Agreed on removing multiple references to Azure Firewall. Updated

Reply:
<img width="1641" alt="image" src="https://github.com/microsoft/Application-Insights-Workbooks/assets/44907284/3254bbe6-f343-4200-b677-25d46fe7c87c">

Reply:
@gumoden i've only really been looking at it for "technical correctness / process", someone from network insights team should really be approving for content/behavior